### PR TITLE
Add serviceBus template

### PR DESCRIPTION
**Pull request by sazeesha**
--------------------MESSAGE FROM ADMIN, DELETE BEFORE SUBMITTING----------------------

Thank you for deciding to contribute to this repository, we're thrilled to have you contribute a template or a fix! :tada: Before we proceed, please be sure to review the contribution guidelines that will make the review process go smoother.
https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md

We've created an outline of recommended sections to fill out that will help make this Pull Request awesome!

--------------------MESSAGE FROM ADMIN, DELETE BEFORE SUBMITTING----------------------

### Changelog
*
*
*
*
*

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@sazeesha__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (sazeesha). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: servicebus-pn-ar/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Create a Service Bus Premium Namespace and AuthorizationRule
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fkafka-on-ubuntu%2Fazuredeploy.json" target="_blank">
```
Comment (singhkay) [inline servicebus-pn-ar/README.md]: These links need to be for this template

File: servicebus-pn-ar/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fkafka-on-ubuntu%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+How to Run the scripts
+----------------------
```
Comment (singhkay) [inline servicebus-pn-ar/README.md]: This section isn't needed. There's an article on how to run these on Azure.com that can be used for this.  Instead in the readme you should add text that explains the scenario that this template accomplishes

File: servicebus-pn-ar/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    "variables": {
+        "location": "[resourceGroup().location]",
+        "ehVersion": "2015-08-01",
+        "defaultSASKeyName": "RootManageSharedAccessKey", 
+        "customSASKeyName": "[concat(parameters('premiumNamespaceName'), concat('/', parameters('namespaceSASKeyName')))]",
+        "defaultAuthRuleResourceId": "[resourceId('Microsoft.ServiceBus/namespaces/authorizationRules', parameters('premiumNamespaceName'), variables('defaultSASKeyName'))]",
+        "customAuthRuleResourceId": "[resourceId('Microsoft.ServiceBus/namespaces/authorizationRules', parameters('premiumNamespaceName'), parameters('namespaceSASKeyName'))]"
+    },
+    "resources": [ 
+	/* Namespace */
```
Comment (singhkay) [inline servicebus-pn-ar/azuredeploy.json]: Comments aren't allowed in JSON

File: servicebus-pn-ar/metadata.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "Create a Service Bus Premium Namespace and AuthorizationRule",
+  "description": "This template creates a Service Bus Premium Namespace and a namespace AuthorizationRule",
+  "summary": "This template creates a Service Bus Premium Namespace and a namespace AuthorizationRule",
+  "githubUsername": "sazeesha",
+  "dateUpdated": "2015-04-06"
```
Comment (singhkay) [inline servicebus-pn-ar/metadata.json]: Date should be updated

**Comment by singhkay** [comment]
closing this PR, merged in #1413

**Pull request closed without merging**